packages/bundle-analyzer/README.md

```diff
@@ -72,6 +72,13 @@ This example shows how the package can be used as a CLI.
 npx @codecov/bundle-analyzer ./dist --bundle-name=test-cli --upload-token=abcd --dry-run
 ```
 
+## Supported Platforms
+
+The CLI tool supports the following operating systems:
+
+- macOS
+- Linux
```

Comment:
after this is released, let's remember to create a GH issue for windows support.

Reply:
good call - create a ticket here! #168